monitorcontrol/monitor_control.py

```diff
@@ -164,13 +164,43 @@ def luminance(self, value: int):
         Args:
             value: new luminance value (typically 0-100)
 
-        Raises:
+        Raises:##### have not implemented or checked
             ValueError: luminance outside of valid range
             VCPError: failed to set luminance in the VCP
         """
         code = vcp.get_vcp_code_definition("image_luminance")
         self._set_vcp_feature(code, value)
 
+
+    @property
+    def contrast(self) -> int:
+        """
+        Gets the monitors contrast.
+
+        Returns:
+            current contrast value
+
+        Raises: ##### have not implemented or checked
+            VCPError: failed to get contrast from the VCP
+        """
+        code = vcp.get_vcp_code_definition("image_contrast")
+        return self._get_vcp_feature(code)
+
+    @contrast.setter
+    def contrast(self, value: int):
+        """
+        Sets the monitors back-light contrast.
```

Comment:
Is contrast tied to the back light?  I have not read the spec on this.

Reply:
It is not contrasts and brightness are two parameters in the protocol